### ISSUE TITLE
suggest add html.zip/pdfs documents for every versions download

### ISSUE THREAD

**Issue by l1t1** [duckdb/duckdb-web#794]
like
https://www.postgresql.org/docs/ did.

**Comment by Mause** [comment]
Can you explain what you see this providing? Are you just wanting to print the documentation?

**Comment by l1t1** [comment]
an offline file is more easy to use in some conditions

**Comment by carlopi** [comment]
There are a few project (also Jekyll based) that have them autogenerated in a 3-4 different of formats (even single page HTML is nice potentially for more control over search).

**Comment by szarnyasg** [comment]
WIP. Partially tackles #794

Important features:
* [ ] Get page-level cross-references right
* [ ] Get section-level cross-references right, e.g. this one should work:
    ```md
    See the [COPY documentation](#..:..:sql:statements:copy#copy-to) for more information.
    ```
* [x] Blog posts should have an absolute URL (starting with `https://duckdb.org/blog`)
* [ ] Add timestamp / version number to the front page
* [ ] Add acknowledgement for using the eisvogel template
* [ ] Add guides
* [ ] Adjust section headers of the original Markdown documents. In practice, this implies that the the ToC should look nice (with no gaps between the headings)
  * [x] Remove duplicate headers #958  

Nice to haves:
* [x] Handle emojis
  * [x] Document the source/licenses of emojis
* [x] ~See whether we can use the `pandoc/latex` Docker image instead of `pandoc/extra`~
    * Not really, many required packages are missing from `pandoc/latex`, e.g. fvextra.sty, scrartcl.cls, scrbase.sty, scrkbase.sty, scrlfile-hook.sty, scrlfile.sty, scrlogo.sty, tocbasic.sty, typearea.sty, lineno.sty
* [ ] Add internals like https://duckdb.org/internals/storage - depends on #938
* [ ] Some tables could look nicer (there are some text overflow issues)
* [ ] Get some of the "raw html" content rendered separately (maybe html->pdf + embed)
* [ ] The railway diagrams are not rendered -> get them rendered. May be tricky as they use JS

**Comment by szarnyasg** [comment]
For browsing a local HTML version of the website, we recommend cloning the repository and building it locally running Jekyll as explained in https://github.com/szarnyasg/duckdb-web/blob/main/building.md.

**Comment by l1t1** [comment]
thanks. 
I am reading the pdf version of 0.9 document now.

**Comment by l1t1** [comment]
p 738~739 ,some code too long to fit in a line
![2023-09-27_102920](https://github.com/duckdb/duckdb-web/assets/23301704/8baed328-7760-4765-8421-615656c743a5)

**Comment by szarnyasg** [comment]
Thanks for reporting. This is something we also noticed but decided to keep it that way as it is only for an illustrative purpose. Users are unlikely to copy the output of this command from the PDF document and use it.
